source/task_scheduler.cpp

```diff
@@ -325,7 +341,7 @@ int TaskScheduler::Init(TaskSchedulerInitOptions options) {
 		auto *const threadArgs = new ThreadStartArgs();
 		threadArgs->Scheduler = this;
 		threadArgs->ThreadIndex = static_cast<unsigned>(i);
-		threadArgs->ThreadStartCallback = options.ThreadStartCallback;
```

Comment:
We *could* get rid of this, and just get the callbacks from the TaskScheduler, like we do in other places. But I'm indifferent. What do you guys think?